src/lib/homestore_backend/tests/homeobj_fixture.hpp

```diff
@@ -138,6 +137,19 @@ class HomeObjectFixture : public ::testing::Test {
         }
     }
 
+    void verify_obj_count(uint32_t num_pgs, uint32_t shards_per_pg, uint32_t blobs_per_shard,
+                          bool deleted_all = false) {
+        uint32_t exp_active_blobs = deleted_all ? 0 : shards_per_pg * blobs_per_shard;
+        uint32_t exp_tombstone_blobs = deleted_all ? shards_per_pg * blobs_per_shard : 0;
+
+        for (uint32_t i = 1; i <= num_pgs; i++) {
```

Comment:
`++i`? probably no-op...

Reply:
I use ++i always, wondered why I did i++ this time, but remembered used github copilot to auto fill which for some reason uses i++ (may be seen previous lines somewhere) :-)